packages/documentation/src/pages/getting-started/common-tasks/updating-formation.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Making Updating Formation and Formataion React 
+tags: formation, formation-react, update, release, publish, npm
+---
+
+## Making updates to formation
+You can find a detailed guide on contributing to formation at [https://dev-design.va.gov/documentation/contributing-to-formation](https://dev-design.va.gov/documentation/contributing-to-formation)
+
+## Making updates to formation-react
+1. Components should be created and tested in vets-website first before being moved over to formation-react
+2. Transfer component to formation-react
+3. Test new node_module component locally. [Check out these instructions on how to do so](/previewing-changes)
+4. Write component tests
+5. [Create your react component documentation](/documentation-guide/doc-page)
+
+## Publishing Module to NPM
+
+After you've tested and previewed your changes locally it's time to publish a new version of the package to NPM.
+
+> Prerequisite: you must be [registered](https://docs.npmjs.com/getting-started/publishing-npm-packages) with NPM, be a member of the "department-of-veterans-affairs" organization, and have the appropriate organization role to publish an update to the module. Verify that you are logged in correctly by running `npm whoami`. If you are unable to publish an update, confirm you are a member of the "department-of-veterans-affairs" organization by going to your Profile page on [npmjs.com](https://www.npmjs.com).
+
+### 1. Submit your PR
+
+Submit a PR that includes all of your code changes. This should include the bump in versions you need, which you can change directly in the package.json files for the modules that are changing. Modules like `formation-react` depend on `formation` via a peer dependency, so you may need to update that as well. We try to keep the peer dependency loose and only update it for breaking changes.
+
+You'll need to choose what type of version update to make:
+
+- `patch` - for bug fixes and minor changes
+- `minor` - for new features that don't break current features or require changes in consuming applications
+- `major` - for backwards breaking changes
+
+> If you are unsure of what to pick, do a major version update on modules that are changing.
+
+### 2. Create a release 
```

Comment:
Shouldn't this step be after step 4?

Reply:
@jbalboni according to GitHub flow process. The release happens right before merging to master. Can you confirm if this is correct?

Reply:
I don't think GH flow matches what we've been doing exactly. We're similar, but we always create releases from code that is in master, but here and in vets-website.

Reply:
Okay, I will update this to reflect what we are already doing.